blog.html

```diff
@@ -114,6 +114,7 @@
                         <a href="grants.html">grants</a>
                         <a href="ideas.html">ideas</a>
                         <a href="mitiq.html">mitiq</a>
+                        <a href="seminars.html">seminars</a> 
```

Comment:
Could we call this "talks" instead of seminars? It looks a bit less intimidating in the menu bar

Reply:
I've updated that everywhere then to talks